packages/strapi-admin/admin/src/components/IntlInput/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { translatedErrors } from 'strapi-helper-plugin';
+import { useIntl } from 'react-intl';
+import { Inputs } from '@buffetjs/custom';
+// import { Col } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+const IntlInput = ({ label: labelId, defaultMessage, error, ...rest }) => {
+  const { formatMessage } = useIntl();
```

Comment:
Why don't we use the `useGlobalContext` here to get the `formatMessage` function? What is the difference between those two functions?

Reply:
Since we have upgraded react-intl we have access to a new hook. I think it will be a good start to stop putting everything in the global context and use the provided hooks instead.